scripts/deploy_docker_to_aws.py

```diff
@@ -63,7 +63,8 @@
     try:
         image = docker_client.images.get(docker_image)
         image.tag(repository=repo_uri, tag=tag)
-        docker_client.images.push(repository=repo_uri, tag=tag)
+        resp = docker_client.images.push(repository=repo_uri, tag=tag)
+        print(resp)
```

Comment:
Do we want an extra string here so we know which response we’re looking at?

(Also, if you do `print(type(resp))`, you can see what the type of `resp` is – I think it’s `requests.Response`, in which case you could use `resp.raise_for_status()` again.)